src/deepsparse/v2/schedulers/scheduler_group.py

```diff
@@ -56,22 +56,12 @@ def submit(
                     **kwargs,
                 )
 
-    def can_process(
-        self,
-        *args,
-        operator: Operator,
-        **kwargs,
-    ) -> bool:
+    def map(self, *args, func: Callable):
```

Comment:
would put this function in the base scheduler instead so all schedulers can share